packages/mobile/src/geth/GethNativeBridgeSigner.ts

```diff
@@ -52,6 +52,12 @@ export class GethNativeBridgeSigner implements Signer {
     addToV: number,
     encodedTx: RLPEncodedTx
   ): Promise<{ v: number; r: Buffer; s: Buffer }> {
+    const { gasPrice } = encodedTx.transaction
+    if (gasPrice === '0x0' || gasPrice === '0x' || gasPrice === '0' || !gasPrice) {
```

Comment:
If a WalletConnect dapp tries to sign a tx without a `gasPrice` are we adding it ourselves? It'd be nice if those don't fail.

Reply:
Yes we're adding it ourselves earlier in the flow. So this won't fail for them.